tldw_Server_API/app/core/AuthNZ/repos/federated_identity_repo.py

```
@@ -0,0 +1,229 @@
+        except Exception as exc:
+            logger.error(f"FederatedIdentityRepo.ensure_tables failed: {exc}")
+            raise
+
+    @staticmethod
+    def _row_to_dict(row: Any) -> dict[str, Any]:
+        if isinstance(row, dict):
+            return dict(row)
+        try:
+            keys = row.keys()
+            return {key: row[key] for key in keys}
+        except Exception as row_keys_error:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)